widgets/GoogleSubscriber-streamingsource.json

```diff
@@ -77,6 +77,20 @@
           "name": "serviceAccountJSON"
         }
       ]
+    },
+    {
+      "label": "Advanced",
+      "properties": [
+        {
+          "widget-type": "number",
+          "label": "Number of Receivers",
```

Comment:
'Receiver' is a Spark term that most users probably won't understand. 'Reader' may be a little better.

Reply:
Can you also update the markdown doc to add information about this setting. Should mention how this relates to the number of executors required to run the pipeline.

Reply:
@albertshau I will also modify the property name from `numberOrReceivers` to `numberOfReaders` for consistency.

Reply:
I have changed the `numberOfReceivers` setting to `numberOfReaders` and updated the docs.